poc/gen_test_vec.py

```diff
@@ -1,19 +1,18 @@
 import json
 import os
 
-from vdaf_poc.common import VERSION, print_wrapped_line
+from vdaf_poc.common import print_wrapped_line
 from vdaf_poc.field import Field128
 from vdaf_poc.idpf import Idpf
 from vdaf_poc.test_utils import gen_test_vec_for_vdaf, test_vec_gen_rand
 from vdaf_poc.xof import Xof
 
 # The path where test vectors are generated.
-TEST_VECTOR_PATH = os.environ.get('TEST_VECTOR_PATH',
-                                  '../test_vec/{:02}'.format(VERSION))
-
+TEST_VECTOR_PATH = os.environ.get('TEST_VECTOR_PATH', '../test_vec/')
```

Comment:
We're dropping the version suffix because we're getting ready for RFC publication?

Reply:
Well, actually it's a little nit-pickier than that: the path is included in the reference to the test vectors, and I didn't want to have to remember to bump the version suffix in the path whenever we bump the version.

Of course, this should hopefully be the last draft with breaking changes🤞.